frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -1188,9 +1188,4 @@ function VirtualKeyboard:scrollDown()
     self.inputbox:scrollDown()
 end
 
-function VirtualKeyboard:clear()
-    logger.dbg("clear input")
-    self.inputbox:clear()
-end
```

Comment:
Looks like this was the only user of InputText:clear() that you removed.
And that currently nothing uses it.
Both were introduced by https://github.com/koreader/koreader/commit/c83f35a0d.
We since replaced it with backspace hold_callback doint: self.keyboard:delToStartOfLine().
So OK.